worker/src/lib/HeliconeProxyRequest/forwarder.ts

```diff
@@ -21,6 +21,8 @@ export async function proxyForwarder(
   ctx: ExecutionContext,
   provider: Provider
 ): Promise<Response> {
+  let organizationId = null;
```

Comment:
can we move this to the lowest possible scope (see next comment)

Reply:
Removed with https://github.com/Helicone/helicone/pull/1174/commits/eae63d83ca5b50a8070bff828897e5120edbdd1d